ksql-common/src/main/java/io/confluent/ksql/properties/with/CommonCreateConfigs.java

```diff
@@ -117,7 +119,14 @@ static void addToConfigDef(
             ConfigDef.Type.STRING,
             null,
             Importance.LOW,
-            "The fully qualified name of the Avro schema to use");
+            "The fully qualified name of the Avro schema to use"
+        )
+        .define(
+            VALUE_DELIMITER_PROPERTY,
+            ConfigDef.Type.STRING,
+            null,
```

Comment:
providing the default of `,` here will likely make the code cleaner.

Reply:
Actually, this might stop you from doing the error handling for non-delimited formats.

Reply:
I don't think we want the default as it would then be inserted for the non delimited formats where it wouldn't be valid to have it.

Reply:
Agreed...